templates/default/kafka.yaml.erb

```diff
@@ -33,7 +33,249 @@ instances:
     <% end -%>
   <% end -%>
 
+<% if @version == 2 %>
+init_config:
+  is_jmx: true
+
+  # Metrics collected by this check. You should not have to modify this.
+  conf:
```

Comment:
Could you include the changes that have been made to the default config on dd-agent/master?

https://github.com/DataDog/dd-agent/blob/5.8.5/conf.d/kafka.yaml.example should have all the latest changes